dawn/main/ansible/Ansible.js

```diff
@@ -16,12 +19,31 @@ const StudentCodeStatus = DawnData.StudentCodeStatus;
 
 const runtimeBuilder = ProtoBuf.loadProtoFile(`${protoFolder}/runtime.proto`);
 const RuntimeData = runtimeBuilder.build('RuntimeData');
+const TCPData = runtimeBuilder.build('TCPData');
+
+let test = 0;
```

Comment:
let declares test to be block scope, and automatically unbinds it when you leave the block.

1. This is global, not a block, so there's not much point of using let.
2. Is unbinding behaviour what you want?